.gitignore

```diff
@@ -4,4 +4,7 @@ Debug/*
 build/*
 *.o
 tax
-tax.exe
\ No newline at end of file
+tax.exe
+tax_details.pdf
+data/*
```

Comment:
this folder has the .dat files for guide.

Hope this won't affect the existing files.

Reply:
Nice catch. Fixed the issue.